core/src/main/java/com/linecorp/armeria/server/DefaultServiceRequestContext.java

```diff
@@ -159,7 +160,7 @@ private DefaultServiceRequestContext(
             long requestStartTimeMicros) {
 
         super(meterRegistry, sessionProtocol,
-              requireNonNull(routingContext, "routingContext").method(), routingContext.path(),
+              requireNonNull(routingContext, "routingContext").method(), routingContext.path(), UUID.randomUUID(),
```

Comment:
Please make `Default{Client,Service}RequestContext` have symmetric and consistent API, i.e. Let a user specify an UUID as a parameter just like you did in `DefaultClientRequestContext`.